src/lib/currency/currency.ml

```diff
@@ -352,6 +352,18 @@ module Make_str (A : Wire_types.Concrete) = struct
 
     let one = Unsigned.one
 
+    let nanomina_unsafe = of_int
+
+    let centimina_unsafe i = of_int (10_000_000 * i)
```

Comment:
the constants are repeated, can they be given re-usable definitions?

Reply:
Done in [fa1e867](https://github.com/MinaProtocol/mina/pull/11835/commits/fa1e8677e19893f51b5df90ec984210415506328)